atc/metric/emitter/prometheus.go

```diff
@@ -38,6 +41,12 @@ type PrometheusEmitter struct {
 	buildsFinishedVec *prometheus.CounterVec
 	buildsSucceeded   prometheus.Counter
 
+	checkBuildsAborted   prometheus.Counter
+	checkBuildsErrored   prometheus.Counter
+	checkBuildsFailed    prometheus.Counter
+	checkBuildsFinished  prometheus.Counter
+	checkBuildsSucceeded prometheus.Counter
+
```

Comment:
Not sure how to handle this comment. I just followed "buildsFinished" metrics:

https://github.com/concourse/concourse/blob/master/atc/metric/emitter/prometheus.go#L34-L37

They are also prometheus-only, but I don't see any comment or doc indicating that.

Reply:
Ugh, that at least makes me feel a bit better about this. Since it looks like we're planning to standardize on prometheus and there's prior precedent for prometheus-only metrics, I'll consider this resolved